docs/ldml/tr35-personNames.md

```diff
@@ -607,23 +612,23 @@ For example:
 3. Pattern C is discarded, because it has the least number of populated name fields.
 4. Out of the remaining patterns A and B, pattern B wins, because it has only 3 unpopulated fields compared to pattern A.
 
-#### 6.5.1 <a name="process-the-namepattern" href="#process-the-namepattern">Choose a namePattern</a>
+#### 6.5.2 <a name="process-the-namepattern" href="#process-the-namepattern">Process a namePattern</a>
 
 If the “winning” namePattern still has fields that are unpopulated in the PersonName object, we process the pattern algorithmically with the following steps:
```

Comment:
Just spotted another problem. 

If the PersonName P has no surname and the pattern has no given name or only a given with initial modifiers, we (logically) create a new PersonName P2 that returns P's given name when asked for the surname, and nothing when asked for the given. (use better wording)

Reply:
Yup.  Totally forgot about that.  Good catch.

Reply:
See how it looks. I inserted a new 6.5.2, and renumbered the old.

Reply:
Forgot to update the TOC; so doing that now.

Reply:
see #2770  (but merge this first). 
I'll drop the numbering once the dust settles on these